index.html

```diff
@@ -54,5 +54,165 @@ <h6>LET'S CONNECT</h6>
         </li>
       </ul>
     </section>
+
+    <section class="work">
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap1.png" alt="" />
+        </a>
+        <ul class="leftblock">
+          <li .primarytext>
+            <h1>Tonic</h1>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+          <li>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </li>
```

Comment:
- [ ] I would suggest you please remove `<li>` tag because we use li tag for listing any object, not for paragraphs.